extensions/camera/non-examples/item.json

```diff
@@ -0,0 +1,13 @@
+{
+  "stac_version": "1.0.0",
+  "type": "Feature",
+  "id": "72360",
+  "geometry": null,
+  "properties": {
+    "datetime": "1952-04-23T00:00:00.000Z",
+    "platform": "Fixed-wing Aircraft",
+    "instruments": ["EAGLE IV"],
+    "mission": "SURVEY_1"
+  },
+  "links": []
+}
```

Comment:
I can't tell from this (or the other) file contents *why* this is not a valid file. Would it be possible to set some property (`title` or `description`?) to be explicit about what's missing/wrong about each file? That way people validating the files can verify that the description corresponds to the error messages they see.